src/test/java/net/minecraftforge/debug/client/model/NewModelLoaderTest.java

```diff
@@ -203,7 +203,7 @@ protected void registerModels()
             withExistingParent(NewModelLoaderTest.item_layers.getId().getPath(), "forge:item/default")
                     .texture("layer0", "minecraft:item/coal")
                     .texture("layer1", "minecraft:item/stick")
-                    .customLoader(ItemLayersModelBuilder::begin)
+                    .customLoader(ItemLayerModelBuilder::begin)
                         .emissive(1)
```

Comment:
This does not look like it is compiling now!
emisive has at least 2 parameters, but needs 3

Reply:
Fixed.  Not sure how that didn't get picked up by my ide